### ISSUE TITLE
Redirects for nightly builds

### ISSUE THREAD

**Issue by favonia** [HoTT/book#1103]
I believe the redirect system set up in #799, #801, and #845 is no longer working. I played a bit with the CI scripts (see #1102) but got stuck because `saunders.phil.cmu.edu` is not working. I wonder what would be the plan to restore the links?

By the way, in addition to all the methods mentioned in #799, we could also put HTML files on GitHub with
```html
<meta http-equiv="refresh" content="0; url=real-file-123456.pdf">
```
to solve the problem. The advantage is that it does not involve other servers and the redirects will always be up to date. The downside is that people might not expect an HTML redirect page before downloading the actual PDF, and tools like `curl` and `wget` do not support this kind of redirects.

Yet another solution is to have a landing page directly on GitHub, say, at `https://hott.github.io/book/download.html`.

**Comment by mikeshulman** [comment]
It would be nice not to need Saunders.

**Comment by favonia** [comment]
It is easy to generate those HTML files with `<meta />` redirection. I wonder how we can make a decision on this.

**Comment by favonia** [comment]
Closed by #1102.

**Comment by favonia** [comment]
Related: #1105.
